doc/feature/restAPI.md

```diff
@@ -0,0 +1,56 @@
+Publishing the REST API exposed by the HTTP Server
+============
+
+The http server exposes C++ objects via a REST API, in a JSON serialized format.
+The purpose of this document is to describe how this API request is implemented,
+and invoked.
+
+## Requirements
+
+* Define how to invoke the application REST API
+* Implement the JSON representation of the REST API
+
+## API
+
+Invoking a GET resquest on the /api url returns a JSON representation of the
+schemas for all registered serializables.
```

Comment:
We previously discussed: ```GET /``` returns list of top-level namespaces, e.g., 'zerobuf', ```GET /zerobuf/render/``` would return the objects in zerobuf::render. ```GET /namespace/object/schema``` would return the schema for the object.